### PR TITLE
pull types allow initial caps

### PR DESCRIPTION
**Pull request by QiWang19**
validate pulltype will allow initial caps form cli or yaml file passed to play kube.

Signed-off-by: Qi Wang <qiwan@redhat.com>

### CODE REVIEW

File: pkg/util/utils.go
Comment (rhatdan) [inline pkg/util/utils.go:558]: This should be "ifnotpresent"

**Review by rhatdan — APPROVED** [review]

File: pkg/util/utils.go
Comment (rhatdan) [reply pkg/util/utils.go:558]: Could you add a test.

**Comment by mheon** [comment]
LGTM once comments from @rhatdan are addressed

/approve

**Comment by rhatdan** [comment]
/lgtm
/hold
@QiWang19 Does this fix an issue?

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by QiWang19** [comment]
> @QiWang19 Does this fix an issue?

close  #7246

**Comment by QiWang19** [comment]
WIP until https://github.com/containers/common/pull/304 merged

**Comment by rhatdan** [comment]
@QiWang19 Containers/common update is available.

File: pkg/util/utils.go
Comment (rhatdan) [inline pkg/util/utils.go:540]: This is a API Breaking change, we can not remove these but we can reroute them.

Comment (rhatdan) [reply pkg/util/utils.go:540]: Take a look at https://github.com/containers/storage/pull/703 to still define these constants and the validate function here, but have them use containers/common.

Comment (QiWang19) [reply pkg/util/utils.go]: fixed

Comment (rhatdan) [reply pkg/util/utils.go]: You just removed the change. I want you to change util.go to use containers/common so the code will be the same.

**Comment by QiWang19** [comment]
@rhatdan @TomSweeneyRedHat @mheon @ashley-cui  PTAL

**Comment by QiWang19** [comment]
still needs fix..

**Pull request closed without merging**
